addons/utm/security/ir.model.access.csv

```diff
@@ -1,7 +1,10 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_utm_campaign_user,access_utm_campaign_user,model_utm_campaign,base.group_user,1,1,1,0
+access_utm_campaign_system,utm.campaign.system,model_utm_campaign,base.group_system,1,1,1,1
 access_utm_medium_user,access_utm_medium_user,model_utm_medium,base.group_user,1,1,1,0
 access_utm_source_user,access_utm_source_user,model_utm_source,base.group_user,1,1,1,0
+access_utm_stage_user,mail.utm.stage,model_utm_stage,base.group_user,1,0,0,0
```

Comment:
I would also give full rights to system for stage and tag. Sub modules will add their own rights definition, but at least system should have them.